bin/ch/ch.cpp

```diff
@@ -94,7 +94,12 @@ void __stdcall PrintUsage()
 
 void __stdcall PrintChVersion()
 {
-    wprintf(_u("%s version %d.%d.%d.0\n"), hostName, CHAKRA_CORE_MAJOR_VERSION, CHAKRA_CORE_MINOR_VERSION, CHAKRA_CORE_PATCH_VERSION);
+#if CHAKRA_CORE_VERSION_RELEASE
```

Comment:
See comments in `ChakraCoreVersion.h` but "RELEASE" configuration is `CHAKRA_CORE_VERSION_RELEASE && !CHAKRA_CORE_VERSION_PRERELEASE`. If `CHAKRA_CORE_VERSION_RELEASE && CHAKRA_CORE_VERSION_PRERELEASE`, then we're in prerelease mode. If both are 0, then we're in dev mode. Both of the latter could be considered `beta` for publication purposes, unless you want to make a distinction between `-dev` and `-preview` (or `-rc` and `-beta`). But IMO `-beta` is fine.